docs/web-player.md

```diff
@@ -65,46 +62,49 @@ import '@liveryvideo/player';
 
 ### HTML
 
-?> Replace the Livery Demo stream id (`5ddb98f5e4b0937e6a4507f2`) below by your own.
-
 ```html
 <livery-player streamid="5ddb98f5e4b0937e6a4507f2"></livery-player>
 ```
 
+?> Replace the Livery Demo stream id (`5ddb98f5e4b0937e6a4507f2`) above by your own.
+
 Please see the [LiveryPlayer](#liveryplayer) API documentation below for additional attributes etc.
 
 ### CSS
 
-To fix the size of the livery-player element for the video etc. to be fit inside, use something like:
+The livery-player element is displayed as a `block`, e.g: using the full width available by default.
```

Comment:
Do you mean i.e.: rather than e.g.: ?

Reply:
I guess so 😅  In Dutch we just have "b.v" (bijvoorbeeld) in both cases.